src/esbmc/bmc.cpp

```diff
@@ -791,10 +791,15 @@ smt_convt::resultt bmct::multi_property_check(
   std::unordered_multiset<std::string> reached_mul_claims;
   bool is_assert_cov = options.get_bool_option("assertion-coverage") ||
                        options.get_bool_option("assertion-coverage-claims");
+  // this might be extended to assert cond
```

Comment:
The number of options for condition coverage is increasing a lot. We should create a new category of options when invoking `esbmc --help`.

Reply:
Will do